explainers/hidden-content-explainer.md

```diff
@@ -271,15 +231,70 @@ only benefit I see to adding support to `window.find` would be to make it easier
 to add WPT tests for beforematch, since we cant have WPTs for find-in-page.
 However, that would also require speccing `window.find`.
 
+## Backwards compatability with the `hidden` attribute
+
+If a browser which doesn't implement this feature renders a `hidden=until-found`
+element, it will still apply the `display:none` style from the `hidden`
+attribute, which will still make the element hidden like the
+`content-visiblity:hidden` style that `hidden=until-found` is supposed to apply.
+This means that websites won't need to specially consider support for this
+feature between browsers, it just means that find-in-page and
+ScrollToTextFragment won't work as nicely in the browsers which don't support
```

Comment:
This isn't true.

```
<style>
.hidden {
  width: 100px;
  height: 100px;
}
</style>
<div hidden=until-found class=hidden></div>
<!-- some other content -->
```

In this case, the box would take up 100x100 space since we're using content-visibility: hidden. We're hiding the content, not the element itself. With display: none, this would occupy 0 space

Reply:
I see...
Should I just change the phrasing to suggest that `hidden=until-found` looks like `hidden` in most but not all cases...?

Reply:
I think it's a big enough distinction that it warrants a separate section. One of the biggest difference between something like `display: none` and `content-visibility: hidden` is its treatment of the element itself that is affected by the property. In `display: none` case, the element itself doesn't have a box so it isn't visible or styled or painted. In `content-visibility: hidden` case, the element itself is painted and drawn. You can check by adding a border on a `content-visibility: hidden` element -- the border is painted and drawn, even though the _contents_ of the element are not.

I feel like this distinction is important enough to point it out separately so that people reading this explainer are very well aware of it.

When it comes to updating this wording, then you can point to that section and say "`hidden` and `hidden=until-found` both hide the content. There, can, however be some visual differences. Link to the section"

Reply:
I updated this section and added the `visibility` mitigation we discussed last week. How does it look now?